ledger-core/virtual/execute/execute_semiintegration_test.go

```diff
@@ -428,7 +423,7 @@ func TestSMExecute_Semi_MethodOnEmptyObject(t *testing.T) {
 			CallFlags:    payload.BuildCallFlags(contract.CallTolerable, contract.CallDirty),
 			CallOutgoing: outgoing,
 
-			Caller:         caller,
+			Caller:         outgoing,
```

Comment:
Caller and CallOutgoing are the same? It's not a typo, is it?

Reply:
typo, will check, but it doesnt matter in this test case

Reply:
fixed